assets/l10n/app_fr.arb

```
@@ -0,0 +1,474 @@
+  "tagCocktail": "Cocktail",
+  "tagTropicalCocktail": "Cocktail tropical",
+  "tagMateTea": "Thé maté",
+  "tagMilk": "Lait",
+  "tagKamaboko": "Kamaboko",
+  "tagOden": "Oden",
+  "tagCheese": "Fromage",
+  "tagEgg": "Œuf",
+  "tagFriedEgg": "Œuf au plat",
+  "tagButter": "Beurre",
+  "done": "Terminé",
+  "cancel": "Annuler",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)